src/components/Calculator.jsx

```diff
@@ -1,66 +1,45 @@
-import React, { Component } from 'react';
+import React, { useState, useEffect } from 'react';
 import Display from './Display';
 import Button from './Button';
 import './Calculator.scss';
 import calculate from '../logic/calculate';
 
-export default class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      calculatorObject: {},
-      display: '',
-    };
-    this.handleCalulatorInput = this.handleCalulatorInput.bind(this);
-    this.updateDisplay = this.updateDisplay.bind(this);
-  }
+export default function Calculator() {
```

Comment:
- [ ] [OPTIONAL] Please use arrow functions since you should use ES6 syntax, I strongly recommend you to take them into account as they can make your code better.